src/explore-education-statistics-common/src/modules/find-statistics/components/charts/MapBlock.tsx

```diff
@@ -1,88 +1,680 @@
+                  value={grouping.selected}
+                  onChange={e =>
+                    this.updateSelectedGroupOption(
+                      grouping,
+                      e.currentTarget.value,
+                    )
+                  }
+                  options={grouping.options}
+                />
+              </div>
+            ))}
+          </form>
+
+          {selected.results.length > 0 ? (
+            <div>
+              {selected.results.map(result => (
+                <div
+                  key={result.id}
+                  className="dfe-dash-tiles__tile govuk-!-margin-bottom-6"
+                >
+                  <h3 className="govuk-heading-m dfe-dash-tiles__heading">
+                    {meta.indicators[result.id].label}
+                  </h3>
+                  <p
+                    className="govuk-heading-xl govuk-!-margin-bottom-2"
+                    aria-label={meta.indicators[result.id].label}
+                  >
+                    <span>
+                      {' '}
+                      {result.value}
+                      {meta.indicators[result.id].unit}{' '}
+                    </span>
+                  </p>
+                  <Details
+                    summary={`What is ${meta.indicators[result.id].label}?`}
+                  >
+                    Description for {meta.indicators[result.id].label}
+                  </Details>
+                </div>
+              ))}
+            </div>
+          ) : (
+            ''
+          )}
+
+          {selected.indicator !== '' ? (
+            <div>
+              <h3 className="govuk-heading-s">
+                Key to {meta.indicators[selected.indicator].label}
+              </h3>
+              <dl className="govuk-list">
+                {legend &&
+                  legend.map(({ min, max, idx }) => (
+                    <dd className={styles.legend} key={idx}>
+                      <span className={styles[`rate${idx}`]}>&nbsp;</span> {min}
+                      {meta.indicators[selected.indicator].unit}&nbsp; to {max}
+                      {meta.indicators[selected.indicator].unit}{' '}
+                    </dd>
+                  ))}
+              </dl>
+            </div>
+          ) : (
+            ''
+          )}
+        </div>
+
+        <div className={classNames('govuk-grid-column-two-thirds')}>
           <Map
-            style={{ height: `${height || 200}px` }}
-            ref={el => {
-              this.mapRef = el;
+            ref={this.mapRef}
+            style={{
+              width: (width && `${width}px`) || '100%',
+              height: `${height || 600}px`,
             }}
+            className={styles.map}
             center={position}
-            // className={styles.map}
             zoom={6.5}
             // minZoom={6.5}
             // zoomSnap={0.5}
             // maxBounds={this.state.maxBounds}
           >
-            {this.renderGeoJson()}
+            {ukGeometry ? (
```

Comment:
This can be simplified to `{ukGeometry && <GeoJSON data={ukGeometry} className={styles.uk} />}`